### ISSUE TITLE
Green borders on android TalkBack not appearing when statusBarColor is set to transparent

### ISSUE THREAD

**Issue by chingjun** [flutter/flutter#33811]
Internal: b/133905887

Customers in Google are experiencing this issue and we managed to get a minimal reproduction of this issue.

## Steps to Reproduce
0. Enable talkback in Android system settings
1. Add the following lines in AndroidManifest.xml under the main activity section:
```
      <meta-data
          android:name="io.flutter.app.android.SplashScreenUntilFirstFrame"
          android:value="true" />
```
2. Add the following lines in the main function of the application:
```
SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle( 
  statusBarColor: Colors.transparent, 
));
```

After these are added, when I click on a widget on the screen, the green border around the widget would not be shown.

**Comment by chingjun** [comment]
@goderbauer

**Comment by matthew-carroll** [comment]
@chingjun I have a couple questions:

1. Which version of Android are you using where this is occurring?

2. Are you sure that the splash screen metadata is related to this? If you remove that metadata and keep everything else the same, does the problem really go away?

**Comment by matthew-carroll** [comment]
@chingjun I tried to repro the issue but I talk back worked fine.

I'm using a Pixel running Android 9.

Can you provide any other guidance to repro?

**Comment by chingjun** [comment]
This issue turns out to only happen internally, closing this issue.

**Comment by chingjun** [comment]
Reopening this issue since the same issue is reproducible with the new Android embedding.

It looks like the new FlutterActivity is also adding a coverView that causes this issue: https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/embedding/android/FlutterActivity.java#L227

Steps to reproduce with new embedding:
1. Switch to use the new FlutterActivity
2. Add the following lines in `main()` to set the status bar to transparent:
```
SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle( 
  statusBarColor: Colors.transparent, 
));
```

**Comment by matthew-carroll** [comment]
Chatted with @mklim offline. We believe the long-term resolution to this issue is to use a `TextureView` instead of a `SurfaceView` in cases where Flutter needs to render on top of other UI elements. The use of a `TextureView` is supported in the new embedding. We are going to leave this ticket open until the new embedding has been adopted, and we have fully removed the z-control that is currently offered in the FlutterView API.

**Comment by matthew-carroll** [comment]
As per my last comment, I'm closing this issue now that the v2 embedding is available on stable. `TextureView` can now be used to bypass this issue.

**Comment by michael-woolsey** [comment]
How does one use this fix?  As of Flutter version 1.22.0 this issue of the missing green TalkBack borders is still present when using:

`SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(statusBarColor: Colors.transparent));`

**Comment by dnfield** [comment]
@michael-woolsey - this bug was close with what is, IMHO, an unsuitable workaround. I've opened #73413 to track a better fix for this.
